doc-tool/src/dotty/tools/dottydoc/staticsite/Site.scala

```diff
@@ -198,7 +200,11 @@ case class Site(
           if (e.kind == "package") ("/index.html", -1)
           else (".html", 0)
 
-        val target = mkdirs(fs.getPath(outDir.getAbsolutePath +  "/api/" + e.path.mkString("/") + suffix))
+        val path = if (scala.util.Properties.isWin)
+          e.path.map(_.replace("<", "_").replace(">", "_"))
+        else 
+          e.path
+        val target = mkdirs(fs.getPath(outDir.getAbsolutePath +  "/api/" + path.mkString("/") + suffix))
```

Comment:
Do we need to change "/" as well?

Reply:
We need to check if `mkdirs()` (and other functions used in `doc-tool`) do support both kinds of file separator. If yes, we can time and extra work (besides keeping the code simpler) !